### DISCUSSION TITLE
so-import-pcap; so-tcpreplay not working on SO 2.3.110 and 2.3.120

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#7906] [Unsupported Versions]
I am using SO Standalone ISO install.  I tested on a new install of version 2.3.120. I also tested on version 2.3.110. I don't see any Suricata alerts on SOC and I do not see any in Kibana. I tried the following three methods:
so-import-pcap 2022-03-01-Emotet-epoch4-infection-with-Cobalt-Strike-and-spambot-traffic.pcap
so-tcpreplay /opt/so/samples/2022-03-01-Emotet-epoch4-infection-with-Cobalt-Strike-and-spambot-traffic.pcap
tcpreplay -v --topspeed -i bond0 2022-03-01-Emotet-epoch4-infection-with-Cobalt-Strike-and-spambot-traffic.pcap

![image](https://user-images.githubusercontent.com/86748904/166997764-fd31c5ee-ff25-40f5-8a68-3c6a3d573d10.png)

![image](https://user-images.githubusercontent.com/86748904/166998092-941203a6-0e42-4b0b-9cb1-6fa716fa9aa3.png)

similar issue: https://github.com/Security-Onion-Solutions/securityonion/discussions/7786

**Comment by TotieBash** [comment]
tag @gbrodt as per request

**Comment by dougburks** [comment]
@TotieBash

Did you install from our Security Onion ISO image or perform a manual installation?

What are the specs of your machine?
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

Does your machine have full Internet access?

Do you have a full NIDS ruleset? What is the output of the following?
```
grep -v "^#" /opt/so/rules/nids/all.rules | wc -l
```
When you import, is it generating logs and alerts in /nsm/import/? What is the output of the following?
```
ls -R /nsm/import
```

**Reply by TotieBash** [reply]
Did you install from our Security Onion ISO image or perform a manual installation?
- I installed using the SO iso image.  I booted from the iso and followed the "standalone" install wizard.  

What are the specs of your machine?
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs
- 12GB RAM; 200GB SSD; 4 CPU CORE
- Installed as virtual machine on ESXi7 Update 3

Does your machine have full Internet access?
- It is installed as "airgapped" config but it currently has internet access.  That is how I was able to get the so-tcpreplay docker image.

Do you have a full NIDS ruleset? What is the output of the following?
- I tested using what came stock with the base ISO image.  I also tested it by updating the latest ET rules.  I manually downloaded the latest ET rules a few days ago and put it in /nsm/repo/rules and ran "so-rules-update".
![image](https://user-images.githubusercontent.com/86748904/167128064-29be7366-0b45-45ec-a297-797cf243a14e.png)


grep -v "^#" /opt/so/rules/nids/all.rules | wc -l
![image](https://user-images.githubusercontent.com/86748904/167127659-c0f26c00-e631-4a16-aa35-eece0972cc84.png)

When you import, is it generating logs and alerts in /nsm/import/? What is the output of the following?
- I think I see Zeek logs from those timestamps but no Suricata alerts.

ls -R /nsm/import
![image](https://user-images.githubusercontent.com/86748904/167128646-1c785a92-4541-4696-9a72-32c94325ca93.png)

**Reply by TotieBash** [reply]
correction: 4vCPU         (not 4 CPU CORE)
![image](https://user-images.githubusercontent.com/86748904/167129756-f169b5e6-4089-400f-8460-9fed785c9015.png)

**Reply by TotieBash** [reply]
Hi Doug I really appreciate you guys responding and looking into it.  I only have less than an hour to reach my lab.  I will be out for two weeks but I will reach back again when I get back.  I say this just in case you need me to provide more screenshot output from my install.  Thanks...

**Reply by TotieBash** [reply]
all suricata eve json file logs are empty.
![image](https://user-images.githubusercontent.com/86748904/167136196-0bcc5f5c-01e5-4afa-a16a-ddafb7326cd9.png)

These are all the PCAP I tested:
![image](https://user-images.githubusercontent.com/86748904/167136373-1c0f2c53-3220-4e9a-895e-60361978350c.png)

**Reply by dougburks** [reply]
> * It is installed as "airgapped" config but it currently has internet access.

Have you tried a fresh installation in non-airgap mode to see if that makes any difference? Perhaps following the screenshots here:
https://docs.securityonion.net/en/2.3/first-time-users.html

**Reply by TotieBash** [reply]
Hello @dougburks I believe my version 2.3.110 is a non-airgap install but I will try it again when I get back.  I pretty much took most of the default for "Standalone" install, and the page where you have to select Wazuh, osquery and two others, I unchecked all of them.  I left the default "IP" for accessing the web interface.  I tested version 2.3.60 last year with the same airgap config and l did pcap import and I dont remember having this issue.

**Reply by dougburks** [reply]
Yeah, I'm trying to get you to a simple config that works so that you can compare to your non-working system and figure out what the problem is. I do import installations all the time and they work fine for me.

**Comment by gbrodt** [comment]
Doug, I can give you remote access through web browser,  to a pod of equipment with the VMs in so you can see it first hand. I just need to get you a username and password.

**Reply by dougburks** [reply]
> I can give you remote access through web browser, to a pod of equipment with the VMs in so you can see it first hand.

@gbrodt Per our previous discussion, we're not able to provide that level of support for free:
https://github.com/Security-Onion-Solutions/securityonion/discussions/7786#discussioncomment-2610162

**Comment by TotieBash** [comment]
@gbrodt looks like found the fix.  I will confirm next week as soon as I get my hands on my lab and report back on the thread.
https://github.com/Security-Onion-Solutions/securityonion/discussions/7786
